pkg/helm/helm_cli_test.go

```diff
@@ -219,6 +219,28 @@ func TestStatusRelease(t *testing.T) {
 	verifyArgs(t, helm, runner, expectedArgs...)
 }
 
+func TestStatusReleaseWithOutputNoFormat(t *testing.T) {
+	expectedArgs := []string{"status", releaseName}
```

Comment:
My reasoning was that, when testing calls to the Helm CLI (or any CLI for that matter) we should always test that the parameters passed in that specific function won't change, and since there's an ìf`in that function, it's always good to test that fork as well. 

I do reckon that there's another extremely similar test for another function, but I wanted to guard myself on this one.

Reply:
I see. The test itself is definitely inline with the others in this file. Happy to keep it. I literally tried to understand your reasoning here, since I feel skeptical about these tests.